templates/components/cart/content.html

```diff
@@ -27,7 +27,9 @@
                     {{#if brand.name}}
                         <p class="cart-item-brand">{{brand.name}}</p>
                     {{/if}}
-                    <h2 class="cart-item-name"><a href="{{url}}">{{name}}</a></h2>
+                    <h2 class="cart-item-name">
+                        <a class="cart-item-name-label" href="{{url}}">{{name}}</a>
```

Comment:
As I know according to BEM cart-item-name is block here and label is element, so we should use cart-item-name__label syntax, agree?

Reply:
to be honest I think that .cart-item-block is block here:)
![cart_table_block](https://user-images.githubusercontent.com/66325265/92603814-6fb10d80-f2b8-11ea-9a82-14bff116e1fe.png)

Reply:
done